webRequest StreamFilter ondata object property clarification (#29843)
mdn/content: files/en-us/mozilla/add-ons/webextensions/api/webrequest/streamfilter/ondata/index.md

{{AddonSidebar()}}

> An event handler called repeatedly when response data is available. The handler is passed an [`event` interface object](/en-US/docs/web/api/event). However, the only reliable content in that object is the `data` property, which includes a chunk of the response data as an {{jsxref("ArrayBuffer")}}.

Reviewer: ```suggestion
An event handler called repeatedly when response data is available. The handler is passed an [`Event` object](/en-US/docs/web/api/event) with a `data` property, which includes a chunk of the response data as an {{jsxref("ArrayBuffer")}}.
```

My statement about reliability was just commentary, doesn't need to be called out.
